system/driver/desktop/window.go

```diff
@@ -70,18 +70,22 @@ func (w *Window) DeActivate() {
 // NewGlfwWindow makes a new glfw window.
 // It must be run on main.
 func NewGlfwWindow(opts *system.NewWindowOptions, sc *system.Screen) (*glfw.Window, error) {
-	_, _, tool, fullscreen := system.WindowFlagsToBool(opts.Flags)
 	// glfw.DefaultWindowHints()
-	glfw.WindowHint(glfw.Resizable, glfw.True)
+	if opts.Flags.HasFlag(system.FixedSize) {
+		glfw.WindowHint(glfw.Resizable, glfw.False)
+	} else {
+		glfw.WindowHint(glfw.Resizable, glfw.True)
+	}
 	glfw.WindowHint(glfw.Visible, glfw.False) // needed to position
 	glfw.WindowHint(glfw.Focused, glfw.True)
 	// glfw.WindowHint(glfw.ScaleToMonitor, glfw.True)
 	glfw.WindowHint(glfw.ClientAPI, glfw.NoAPI)
 	// glfw.WindowHint(glfw.Samples, 0) // don't do multisampling for main window -- only in sub-render
+	fullscreen := opts.Flags.HasFlag(system.Fullscreen)
 	if fullscreen {
 		glfw.WindowHint(glfw.Maximized, glfw.True)
```

Comment:
Fullscreen is doing two things at once: maximized and actual fullscreen, which need to be separated; maximized non-fullscreen is a key modality on Windows.